lib/msal-browser/src/cache/ITokenCache.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ServerAuthorizationROPCResponse } from "@azure/msal-common";
+import { SilentRequest } from "../request/SilentRequest";
+import { LoadTokenOptions } from "./TokenCache";
+
+export interface ITokenCache {
+
+    /** API to side-load tokens to MSAL cache */
+    loadTokens(request: SilentRequest, response: ServerAuthorizationROPCResponse, options: LoadTokenOptions): void;
```

Comment:
Admittedly not the biggest fan of this name, will stew on this.

Reply:
Yeah, definitely open to suggestions on any of the naming.

Reply:
The filename stays right? `TokenCache` is a public interface for msal-node. For uniformity I think it helps to have similar interface implementations named the same way. Function names, yes, there could be more meaningful naming I guess.

Reply:
"loadTokens" is the part I was referring to.

Reply:
How about `sideloadTokens`? A bit more accurate and descriptive?

Reply:
Thanks for the input, all. Have changed the API name to `loadExternalTokens` for uniformity with `ExternalTokenResponse`. Let me know if there are any other concerns.